client/lib/plans/constants.js

```diff
@@ -438,11 +448,13 @@ export const PLANS_LIST = {
 			FEATURE_PREMIUM_SUPPORT,
 		],
 		getSignupFeatures: () => [
+			FEATURE_OFFSITE_BACKUP_VAULTPRESS_DAILY,
 			FEATURE_PREMIUM_SUPPORT,
-			FEATURE_BACKUP_STORAGE_SPACE_UNLIMITED_SIGNUP,
+			FEATURE_SPAM_AKISMET_PLUS,
 			FEATURE_ALL_FREE_FEATURES,
 		],
 		getBillingTimeFrame: () => i18n.translate( 'per year' ),
+		getSignupBillingTimeFrame: () => 'per year', //PLANS A/B TEST: Translate if test passes
```

Comment:
Any good reason to not wrap this one in a `translate` call immediately? This string is already translated in the above line.